spl-tokens/app/app/LICENSE.md

```
@@ -0,0 +1,21 @@
+MIT License
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)